src/styles/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $cardWidth;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $BorderColor;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: auto;
+  }
+
+  &__title {
+    margin-top: 40px;
+    margin-bottom: 0;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $blackColor;
+    overflow: hidden;
+  }
```

Comment:
Not sure about `overflow` here, can you explain the need for it ?

Reply:
Its really unnecessary, we don't need it here)